recipes/mdal-python/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
```

Comment:
Is this actually needed?
```suggestion
```
If it is you should use `$target_platform` instead of `uname` to make cross-compilation possible.